admin/assets/stylesheets/elements/_hdf.scss

```diff
@@ -0,0 +1,21 @@
+// Smaller inset checkboxes
+.multiple-choice.inset {
+  margin-left: 50px !important;
+
+  input[type=checkbox] + label {
```

Comment:
I am not sure why we would need custom css, why doesn't the standard markup cover the design?
https://govuk-elements.herokuapp.com/form-elements/

Reply:
Needed to add styles for a inset list of checkboxes because the standard gov uk elements don’t have any styles for it. Also, the checkboxes on the prototype are smaller than the standard gov uk size.

Reply:
@danwilliams - I question both of these statements.  Why are we using non-standard sizes?   We should be using standard sizes, surely?  Perhaps we should discuss?

Reply:
Okay - I've seen the prototype and now understand that GDS does not have a pattern for inset radio boxes.